webapp/graphite/render/functions.py

```diff
@@ -4629,6 +4643,7 @@ def group(requestContext, *seriesLists):
 group.params = [
   Param('seriesLists', ParamTypes.seriesList, multiple=True),
 ]
+group.aggregator = True
```

Comment:
This also doesn't make much sense as an aggregator, you'd end up with the value of the first series in each group.